### PR TITLE
updated lib and exposed more configs

### PR DESCRIPTION
**Pull request by disruptivepatternmaterial**
so one big change is the flip in the colors, you might not want to accept that...the next step is to make that flip a config I can send in from esphome

I am seeing from 40-75 frames / second and other improvements.

### CODE REVIEW

**Comment by disruptivepatternmaterial** [comment]
oh and I included my clock yaml

**Comment by sekureco42** [comment]
Thank you very much for your contribution. I will merge them as soon as you have reviewed my comments I made on several lines of code. Could you review and adjust them please?

**Comment by disruptivepatternmaterial** [comment]
sure...you clearly know more than I do about this...so just trying to make it work...I also added in a flag (rgb_order) today because some of my panels are RGB and some are RBG. Once I have that all working I'll submit that too.

**Comment by disruptivepatternmaterial** [comment]
> Thank you very much for your contribution. I will merge them as soon as you have reviewed my comments I made on several lines of code. Could you review and adjust them please?

ok maybe I am being dense, but where did you put the comments? I looked at the files and the comments/conversation part and I am not seeing anything.

File: components/hub75_base/hub75_display.h [hunk 1]
```diff
@@ -38,8 +39,8 @@ class HUB75Display : public PollingComponent, public display::DisplayBuffer {
 #endif  // VERSION_CODE(2023, 12, 0)
   public:
     virtual void setup();
```
Comment (disruptivepatternmaterial) [inline components/hub75_base/hub75_display.h:41]: test

**Comment by disruptivepatternmaterial** [comment]
so I am closing this one...I merged in your changes, fix a few things, and added in the RGB/RBG flag, which I tested on my panels.

**Pull request closed without merging**